.github/workflows/maven.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Maven
+on: 
+  push:
+    branches-ignore:
+     - documentation
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: '11'
```

Comment:
Hmm.. currently I provide even very old eclipse versions with the plugin and support normally min JDK8. 

There was even an issue where a user wanted still support for eclipse photon... see https://github.com/de-jcup/eclipse-asciidoctor-editor/issues/316 

Do you think it's a good idea to go to SDK 11 here?

Reply:
Having no option to comment on the deleted gradle.yml workflow: As long as the build with maven does not work and "normal" tests are not run by maven as well  I would keep the old gradle build - it's generated badges are used inside the README in the root folder as well - so the README must be changed after full maven support as well to use the correct github action badge.

So please revert the delete - this time.

Reply:
As we move this to a branch, we should IMHO still delete the gradle file if it fails the verfication. I test this.

Reply:
Java 11 is required by Tycho since a while. It still can build plug-ins using BREE smaller than Java 11.